### PR TITLE
Add command to force hyphens to appear.

### PR DESCRIPTION
**Pull request by rpspringuel**
Addresses [task #7767 on the old tracker](https://gna.org/task/?7767)

### CODE REVIEW

**Comment by eschwab** [comment]
Does `maximumspacewithoutdash` reset after a score? If not, it may be useful to add an accompanying macro to return to the default value. `\greautohyphen` or `\gredefaulthyphen`

**Comment by rpspringuel** [comment]
Done.

File: doc/Command_Index_User.tex [hunk 1]
```diff
@@ -155,6 +154,20 @@ \subsubsection{Including scores}
   \gabcsnippet[0]{(c3) Al(eg~)le(gv.fhg)lu(efe___)ia(e.) (::)}
 \end{latexcode}
 
+\macroname{\textbackslash greforcehyphen}{}{gregoriotex.tex}
+Tell Gregorio\TeX\ to force the appearance of hyphens between all
+syllables in polysyllabic words.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
+
+Note: This command only affects the hyphens within a score.  Hyphens
```
Comment (eroux) [inline doc/Command_Index_User.tex]: I think this not is unnecessary, as the opposite would just be a bug, not a reasonable design choice...

File: tex/gregoriotex-main.tex [hunk 2]
```diff
@@ -1378,6 +1378,15 @@
   \global\let\grehyph\grezerhyph %
 }%
 
+% macro to force hyphenation of all syllables.
+\def\greforcehyphen{
+  \global\grechangedim{maximumspacewithoutdash}{-16383.99999pt}{0}
+}
+
+% macro to force hyphenation of all syllables.
```
Comment (eschwab) [inline tex/gregoriotex-main.tex]: Copy paste oops. Macro un-forces hyphenation.

**Comment by rpspringuel** [comment]
Fixed.

**Comment by eroux** [comment]
Ok for me!

**Pull request merged**